Makefile

```diff
@@ -1,10 +1,10 @@
-TAG="1.0.0"
+TAG="1.0.1"
```

Comment:
maybe 1.1.0 as you've added new features?

Reply:
that's right, fixed